inferno-framework/inferno-framework.github.io
# Update Documentation Section

# Summary

Updated the Documentation section for content and clarity. This mostly involved page reorganization and updating content on each page for grammar/clarification. See commit messages for more specifics.

# Testing Guidance

- Make sure that there is nothing in the [current Documentation Site](https://inferno-framework.github.io/inferno-core/writing-tests/properties.html) that is missing here. 
- Make sure there is nothing stated in the Documentation Site that is explicitly wrong.
- Any major changes, such as adding a tutorial to the site, is out of scope. We are looking for parity with the old site. If you notice a major change that should be on the Documentation site, make a Jira ticket and add it to the "Review Inferno Documentation Site" epic.
